plat/hisilicon/hikey960/hikey960_pm.c

```diff
@@ -33,26 +33,9 @@ static uintptr_t hikey960_sec_entrypoint;
 
 static void hikey960_pwr_domain_standby(plat_local_state_t cpu_state)
 {
-	unsigned long scr;
-	unsigned int val = 0;
-
-	assert(cpu_state == PLAT_MAX_RET_STATE);
-
-	scr = read_scr_el3();
-
-	/* Enable Physical IRQ and FIQ to wake the CPU*/
-	write_scr_el3(scr | SCR_IRQ_BIT | SCR_FIQ_BIT);
-
-	set_retention_ticks(val);
+	/* Add barrier before CPU enter WFI state */
+	dsb();
```

Comment:
The SCR routing bit manipulation is still required I think to ensure interrupt can wakeup the CPU.
```
scr = read_scr_el3();
/* Enable Physical IRQ and FIQ to wake the CPU*/
write_scr_el3(scr | SCR_IRQ_BIT | SCR_FIQ_BIT);
wfi();
write_scr_el3(scr);
```

Reply:
@Leo-Yan, If you can rework this comment, we can merge the PR.

Reply:
Hi @soby-mathew, I have updated this patch about one or two days ago, but I found my comment seems have not send out. Could you check again if you could see the updated patch for following your previous comment?

Reply:
Ah, yes, I see your change now. Thanks. Github UI is sometimes confusing.